styles.css

```diff
@@ -0,0 +1,145 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,600&display=swap');
+
+:root {
+  --pry-font-family: 'Poppins', sans-serif;
+  --pry-font-style: normal;
+  --pry-color: #6070ff;
+  --sec-color: #172b4d;
+  --tertiary-color: #344563;
+  --pry-bgcolor: #e5e5e5;
+  --sec-bgcolor: #fff;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #e5e5e5;
+}
+
+.headers {
+  background-color: var(--sec-bgcolor);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 3.625rem 1.8rem 0.875rem 1.5rem;
+}
+
+.mylogo {
+  font-family: var(--pry-font-family);
+  font-style: var(--pry-font-style);
+  font-weight: 700;
+  font-size: 1.125rem;
+  line-height: 1.25rem;
+}
+
+.mylogo,
+.nav-menu .fas {
+  color: var(--pry-color);
+  text-align: center;
+}
+
+.nav-menu .fas {
+  align-self: center;
+}
+
+.about-me {
+  background-color: var(--sec-bgcolor);
+  border-bottom-left-radius: 70px;
+  height: 98.1vh;
+}
```

Comment:
- [ ] The about-me section has the proper background image.
- [ ] The content should aligned with flexbox.